babel.dependencies.config.js

```diff
@@ -13,6 +13,7 @@ const config = {
 				exclude: [ 'transform-typeof-symbol' ],
 			},
 		],
+		'@babel/preset-typescript',
```

Comment:
This config is only for transpiliing dependencies (ie stuff in node_modules). We shouldn't need to deal with typescript there.